pyocd/target/builtin/target_STM32F767xx.py

```diff
@@ -0,0 +1,190 @@
+# pyOCD debugger
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Can you please add your copyright here? (Technically should be added to every file you touch. But a file needs at least one copyright statement.)

Reply:
I'm not good at legal things: Who should be copyright holder?
I don't see any private copyrights notices in source code... does it mean that there is no private contributors to pyOCD ?

Reply:
Depends. If you're doing this on your own time, use your name as the copyright holder. If it's on employer time, use the company name.

You're right, there aren't private copyrights, but that's mostly a failure on the reviewers' part. I'm trying to improve on it. There have been plenty of private contributions, but in the past the contributor rules were different and I think work was assigned to Arm. Even my own contributions from before I joined Arm were assigned to Arm.

Reply:
It's a private contribution - professionally I'm ML engineer, but sometimes I miss low level programming ;)

Thanks for making the project more friendly to open source community!